server/src/engine.rs

```diff
@@ -0,0 +1,87 @@
+//! Implementation of pricing logics based on the ride type.
+
+use crate::grpc::PricingRequest;
+
+/// Get pricing for a given query
+///
+/// # Arguments
+/// * `request` - PricingRequest
+pub fn get_pricing(query: PricingRequest) -> f32 {
+    match query.service_type {
+        0 => get_cargo_pricing(query),
+        1 => get_rideshare_pricing(query),
+        2 => get_charter_pricing(query),
+        _ => 0.0,
+    }
+}
+
+// ------------------------------------------------------------------
+// Cargo pricing assumptions
+// Expect these constants to be pulled from svc-storage in the future
+// https://docs.google.com/spreadsheets/d/1mjPtaIn3E5m7r4nyKt_sJKG9BSFm2ty7Gzo7OqERxwo
+// ------------------------------------------------------------------
+
+/// Take off and landing cost in dollars.
+const CARGO_TOL_COST: f32 = 2.8;
+/// Cruise speed in kilometers per hour.
+const CARGO_CRUISE_SPEED: f32 = 240.0;
```

Comment:
add _KPH to end of constant

Always put the units in the variable name. Same for the following variables

_KWH

_RATE_USD_HR 

etc.

https://www.simscale.com/blog/nasa-mars-climate-orbiter-metric/

Reply:
[f41081e](https://github.com/Arrow-air/svc-pricing/pull/1/commits/f41081e6863d0026fbd06843877e87793d225678)